src/main/steps/common/keep-details-private/KeepYourDetailsPrivateMapper.ts

```diff
@@ -1,65 +1,76 @@
 /* eslint-disable @typescript-eslint/explicit-module-boundary-types */
 import { CaseWithId } from '../../../app/case/case';
-import { ConfidentialityList, KeepDetailsPrivate, PartyDetails, YesOrNo } from '../../../app/case/definition';
+import { ConfidentialityList, PartyDetails, YesOrNo } from '../../../app/case/definition';
 import type { AppRequest } from '../../../app/controller/AppRequest';
 
-export const setKeepYourDetailsPrivate = (partyDetails: PartyDetails, req: AppRequest): PartyDetails => {
-  let keepDetailsPrivate: KeepDetailsPrivate;
+export const prepareKeepDetailsPrivateRequest = (req: AppRequest): Partial<PartyDetails> => {
```

Comment:
Why do we need the entire req , just pass the usercase to this method as argument.

Reply:
Changed this to take usercase as the argument